lib/matplotlib/contour.py

```diff
@@ -1409,13 +1409,13 @@ def find_nearest_contour(self, x, y, indices=None, pixel=True):
 
         Returns
         -------
-        contour : `.Collection`
-            The contour that is closest to ``(x, y)``.
+        path : int
+            The index of the path that is closest to ``(x, y)``.
         segment : int
-            The index of the `.Path` in *contour* that is closest to
+            The index within that closest path of the segment that is closest to
```

Comment:
I'm using "segment" here to mean one of the pieces you get when you split the path by its MOVETOs.  I'm not sure I have the language right though.  What exactly defines a segment?

Reply:
connected component, I think

Reply:
So should I change “segment” to “connected component” here?  Or something else?

Reply:
I guess it could be a 'subpath'?

Reply:
OK I have changed it to "subpath" and also added some explanation of what paths and subpaths correspond to in the plot.